experimental/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ For notes on migrating to 2.x / 0.200.x see [the upgrade guide](doc/upgrade-to-2
 ### :rocket: Features
 
 * feat(instrumentation-http): capture synthetic source type on requests [#5488](https://github.com/open-telemetry/opentelemetry-js/pull/5488) @JacksonWeber
+* feat(instrumentation-grpc): support migration to stable HTTP semconv [#9659](https://github.com/open-telemetry/opentelemetry-js/pull/9659) @JamieDanielson
```

Comment:
Should be PR number 5653 right? Not sure where 9659 came from.

Reply:
no idea!